jekyll/_cci2/artifacts.md

```diff
@@ -16,18 +16,18 @@ This document describes how to work with Artifacts in the following sections:
 
 ## Artifacts Overview
 
-Artifacts persist data after a {% comment %} TODO: Job {% endcomment %}Workflow is completed and may be used for longer-term storage of the outputs of your build process. For example, a Java project that produces a `.jar` file. When the Java build/test process passes, the output of the process (the `.jar` file) can be stored as an artifact. The `.jar` file is available to download from the CircleCI artifacts system long after the workflow that created it has finished.
+Artifacts persist data after a job is completed and may be used for longer-term storage of the outputs of your build process. For example, a Java project that produces a `.jar` file. When the Java build/test process passes, the output of the process (the `.jar` file) can be stored as an artifact. The `.jar` file is available to download from the CircleCI artifacts system long after the workflow that created it has finished.
 
 ![artifacts data flow]( {{ site.baseurl }}/assets/img/docs/Diagram-v3-Artifact.png)
 
 Another example of an Artifact is a project that is packaged as an Android app where the `.apk` file is uploaded to Google Play. 
 
-If your {% comment %} TODO: Job {% endcomment %}build produces persistent artifacts such as screenshots, coverage reports, core files, or
+If a job produces persistent artifacts such as screenshots, coverage reports, core files, or
 deployment tarballs, CircleCI can automatically save and link them for you.
 
 ![artifacts tab screeshot]( {{ site.baseurl }}/assets/img/docs/artifacts.png)
 
-Find links to the artifacts at the top of the {% comment %} TODO: Job {% endcomment %}Build page.
+Find links to the artifacts at the top of the **Jobs page**.
```

Comment:
Should this be a link? (It'll still be `/dashboard`)

Reply:
Yes, it should! Great addition.

Reply:
Actually, you know what — in most of these cases, we're talking about a **specific** build page. So the dashboard link won't actually display that Artifacts tab that also keeps getting referenced.

Reply:
Ah, good point

Reply:
This shouldn't be plural. The original change is correct.